pageserve/templates/base.html

```diff
@@ -20,39 +20,55 @@
     <title>{% block title %}{% endblock title %} - Large Events</title>
     <meta name="google-signin-scope" content="profile email">
     <meta name="google-signin-client_id" content="{{ app_config['GAUTH_CLIENT_ID'] }}">
-    <script src="https://apis.google.com/js/platform.js" async defer></script>
+    <script src="https://apis.google.com/js/platform.js?onload=onLoad" async defer></script>
+    <script>
+        // initialize gapi.auth2 to make sign out button work
+        function onLoad() {
+            gapi.load('auth2', function () {
+                gapi.auth2.init();
+            });
+        }
+    </script>
 </head>
 
 <body>
     <header>
         {% block header %}{% endblock header %}
     </header>
     {% block login_buttons %}
-    {% if not signed_in %}
```

Comment:
Where is signed_in passed to the template? Or does it automatically have the info?